src/main/java/com/rsakin/vocid22tracker/repo/HospitalRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.rsakin.vocid22tracker.repo;
+
+import com.rsakin.vocid22tracker.model.entity.Hospital;
+import org.springframework.data.repository.CrudRepository;
+
+public interface HospitalRepository extends CrudRepository<Hospital, Long> {
```

Comment:
This repo is simply used for the initial data loading, correct?

Reply:
Yes, right